Assets/Code/VehicleEditor/EditSystem.cs

```diff
@@ -141,17 +192,34 @@ partial class EditSystem : SystemBase {
         return false;
     }
 
-    void PlacePart(ref EntityCommandBuffer actionQueue, Entity part, LocalTransform placeTransform, Entity parentPart) {
+    void PlacePart(ref EntityCommandBuffer actionQueue, Entity part, LocalTransform placeTransform) {
         var newPart = EntityManager.Instantiate(part);
 
         // todo - place one part at a time + modifier to place more at a time
 
-        if (EntityManager.Exists(parentPart)) {
-            actionQueue.AppendToBuffer(parentPart, new PartChildBuffer { Value = newPart });
-        }
         actionQueue.AddBuffer<PartChildBuffer>(newPart);
-
         actionQueue.SetComponent(newPart, placeTransform);
+        
+        /*if (parentPart != Entity.Null) // this is not needed since there is no more parent (seperated into to methodes) (might wanna merge them back into one in the future)
```

Comment:
does deleting part trees still work?

Reply:
Deleting Part Tree still works but it might need some changes :
- The part which is deleted should be erased from his siblings to Entity.Null
- And I'm not sure if you already did this but it might be good to delete that part from the buffer of children of the parent of deleted part